src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -534,16 +534,16 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Proc
             yield return new FormatViewDefinition("process",
                 TableControl.Create()
                     .AddHeader(Alignment.Right, label: "NPM(K)", width: 7)
-                    .AddHeader(Alignment.Right, label: "PM(K)", width: 8)
-                    .AddHeader(Alignment.Right, label: "WS(K)", width: 10)
+                    .AddHeader(Alignment.Right, label: "PM(M)", width: 8)
+                    .AddHeader(Alignment.Right, label: "WS(M)", width: 10)
                     .AddHeader(Alignment.Right, label: "CPU(s)", width: 10)
                     .AddHeader(Alignment.Right, width: 6)
                     .AddHeader(Alignment.Right, width: 3)
                     .AddHeader()
                     .StartRowDefinition()
                         .AddScriptBlockColumn("[long]($_.NPM / 1024)")
-                        .AddScriptBlockColumn("[long]($_.PM / 1024)")
-                        .AddScriptBlockColumn("[long]($_.WS / 1024)")
+                        .AddScriptBlockColumn("\"{0:N3}\" -f [float]($_.PM / (1024 * 1024))")
```

Comment:
Yes, changing `1024 * 1024` to `1MB` would be better. I didn't know that before.
As for the precision, if we use `:N2` then for processes which take up 1-9KB of memory, it would be shown as `0.00`, whereas using `:N3` would show, for example, `0.005`, which is the correct value.